DomainContent/Rust/DJ-Tools/DJ-Tools_Spawner.js

```diff
@@ -0,0 +1,560 @@
+(function () {
+    // Init
+    var PHLASH_TABLE_NAME = "Set_Phlash_Tables",
+        baseURL = "https://hifi-content.s3.amazonaws.com/milad/ROLC/Organize/Projects/Hifi/Scripts/DJ-Tools/",
+        particlePadLeftModel = "https://hifi-content.s3.amazonaws.com/alan/dev/particle-pad-1.fbx",
+        particlePadRightModel = "https://hifi-content.s3.amazonaws.com/alan/dev/particle-pad-2.fbx",
+        particleScriptServer = baseURL + 'DJ_EndPoint_Particle_Server.js',
+        lightScriptServer = baseURL + 'DJ_EndPoint_Light_Server.js',        
+        zoneScript = baseURL + 'DJ_Input_Zone_Client.js',
+        DEBUG = true,
+        LEFT = "Left",
+        RIGHT = "Right",
+        GROUP_LEFT = "Group_Left",
+        GROUP_RIGHT = "Group_Right",
+        DEBUG_CUBE = "debugCube",      
+        INPUT = "input",
+        ENDPOINT = "endPoint";
+
+    // Polyfill
+    Script.require(Script.resolvePath("./Polyfills.js"))();
+
+    // Colections
+    var phlashProps = gProps(PHLASH_TABLE_NAME),
+        allEnts = [],
+        particleBaseProps = {
```

Comment:
Use the same formatting in JSON property lists here and below - some are in quotes and some not. Either one works, just should be consistent.

Reply:
done